### DISCUSSION TITLE
Welcome to sdupdates Discussions!

### DISCUSSION THREAD

**Discussion by questianon** [questianon/sdupdates#1] [Announcements]
## Welcome to sdupdates 
This is a place where you can discuss anything relating to SD or sdupdates, like:
- Information that's missing/new
- Models/embeddings/hypernetworks that are missing
- Questions 
- General discussion about SD

**Comment by lionrooter** [comment]


**Reply by questianon** [reply]
Thank you for all the kind words 😄 My only socials for right now are listed in my bio, but I'll probably end up making more.

**Comment by ChanseyIsTheBest** [comment]


**Reply by questianon** [reply]
Yea definitely, I'll add it right now. Thanks for the link and training info

**Comment by Linaqruf** [comment]
Hi, I uploaded AnythingV3.0.ckpt (Chinese NAI Resume Training Model) to Huggingface, I also pruned it to run on 4G. Do you think you could add it to the list?

In this repository there are 3 files, unpruned model, pruned model, and VAE 
https://huggingface.co/Linaqruf/anything-v3.0/tree/main

Maybe pickled, I'm still learning how to unpickle it.

Here's a comparison between using animefull-finalpruned.ckpt vs AnythingV3.0-pruned.ckpt, I'm using latest pull of Automatic1111's Stable Diffusion Web UI to generate both images.

![image](https://user-images.githubusercontent.com/50163983/200586422-9f2d8fc8-6478-4ba8-a173-cd9ae93f6733.png)

Thank you!

**Reply by USBhost** [reply]
> Hi, I uploaded AnythingV3.0.ckpt (Chinese NAI Resume Training Model) to Huggingface, I also pruned it to run on 4G. Do you think you could add it to the list?
> 
> In this repository there are 3 files, unpruned model, pruned model, and VAE https://huggingface.co/Linaqruf/anything-v3.0/tree/main
> 
> Maybe pickled, I'm still learning how to unpickle it.
> 
> Here's a comparison between using animefull-finalpruned.ckpt vs AnythingV3.0-pruned.ckpt, I'm using latest pull of Automatic1111's Stable Diffusion Web UI to generate both images.
> 
> ![image](https://user-images.githubusercontent.com/50163983/200586422-9f2d8fc8-6478-4ba8-a173-cd9ae93f6733.png)
> 
> Thank you!

Where was the source for that one? Was it in the original NAI leak?

**Reply by Linaqruf** [reply]
The left one is from the original nai leak, pruned model. The right one is on sdupdates2#newsfeed today, 11/8+11/7, looks like the chinese community did resume training on the original nai leak so you can get more detailed results.

**Reply by zestybaby** [reply]
> The left one is from the original nai leak, pruned model. The right one is on sdupdates2#newsfeed today, 11/8+11/7, looks like the chinese community did resume training on the original nai leak so you can get more detailed results.

Is there a link to the original post? I can read Chinese and I'm interested in this project, but can't find the OP, maybe the name is too generic.

**Reply by aliencaocao** [reply]
https://www.bilibili.com/read/cv19603218
I found this @zestybaby

**Reply by questianon** [reply]
Hi, thanks for making this, I'll add it in right now

**Comment by R-N** [comment]
Hello, can we maybe organize a Google Drive repo for the usual models such as SD, WD, etc? Not sure about the licenses and permissions though. 

In Google Drive, we can add a shortcut of someone's file into our drive, and it won't take space. The shortcut can be used in Google Colab just like normal file. I was doing this with SD 1.4 but SD 1.5 doesn't have a Google Drive link.

**Reply by questianon** [reply]
Hi, yea that sounds like a good idea for people who want to use colab but don't want to wait to upload the model every time. It also gives people another mirror to download from if they dislike torrenting/mega/whatever other mirrors there are. The only issue might be takedowns? (like NAI), though I'm not too sure how that works so take this with a grain of salt.

I'll be sure to add in the repo if someone creates it. If no one wants to, I can create it sometime in the future

**Comment by Linaqruf** [comment]
Just found this when surfing on twitter, maybe it's what we called English Grimoire, I think it's useful for people who don't understand Japanese, Chinese, and Korean spellbook

https://lunarmimi.net/freebies/novelai-anime-girl-prompt-guide/

**Reply by questianon** [reply]
Sorry for the late reply, I was busy for the past few days. I'll add it in, but the link seems to have died. Thanks for linking it though

**Reply by questianon** [reply]
Nevermind, it works now. Thanks again

**Reply by USBhost** [reply]
> Sorry for the late reply, I was busy for the past few days. I'll add it in, but the link seems to have died. Thanks for linking it though

Seems to work for me

**Comment by CapsAdmin** [comment]
I made an embedding for stable diffusion 2.0 based on greg rutkowski's work. The idea is to illustrate that you can get close to his style just from textual inversion despite his artwork not existing in the training data. This was the case with the 1.5 models as well, except the previous clip model was trained on him.

https://www.reddit.com/r/StableDiffusion/comments/z5hlqa/greg_rutkowski_embedding_for_automatic1111/

**Reply by questianon** [reply]
I'll add this when organizing the embeddings section, thanks for making this (and sorry for the late reply)

**Comment by Linaqruf** [comment]
Hi! I write a colab notebook based on [Japanese Fine-tuning Guide](https://note.com/kohya_ss/n/nbf7ce8d80f29#c9d7ee61-5779-4436-b4e6-9053741c46bb) that can run in 12GB VRAM , by [Kohya_ss](https://twitter.com/kohya_ss), it's called Kohya Trainer. 

This might be a good choice for everyone who wants to be an AI trainer but doesn't get enough VRAM
LINK : https://github.com/Linaqruf/kohya-trainer/blob/main/kohya-trainer.ipynb

Do you think you could add it to the list?

**Reply by Linaqruf** [reply]
I also got permission from [Kohya_ss](https://twitter.com/kohya_ss), to be able to publish this notebook (if anyone asking about am i stealing someone code):

![image](https://user-images.githubusercontent.com/50163983/204123629-5c646a0d-021d-432a-8cba-061128fe067d.png)

**Reply by Linaqruf** [reply]
It runs well so far, I use it to fine-tune a model and create Hitokomoru Diffusion and other experimental model, it already available on huggingface if anyone interested


https://huggingface.co/Linaqruf/hitokomoru-diffusion
![image](https://user-images.githubusercontent.com/50163983/204123732-daf9e570-b6e3-448d-81c8-900f8dc5cd29.png)

**Reply by Linaqruf** [reply]
Thank you in advance!

**Reply by questianon** [reply]
Ah interesting, I'll be sure to add this and add proper credit when organizing the Training section. Thanks for linking and making this (and sorry for the late reply).
